packages/driver/test/cypress/integration/commands/actions/focus_spec.coffee

```diff
@@ -153,6 +153,14 @@ describe "src/cy/commands/actions/focus", ->
       cy.get("[data-cy=rect]").focus().then ->
         expect(onFocus).to.be.calledOnce
 
+    it "can focus on readonly inputs", ->
```

Comment:
Because this is **the test** that is effectively testing this entire PR - it needs a link above the issue to the issue that it's fixing.